src/Domain/Metrics/Code/Functions.php

```diff
@@ -42,7 +42,7 @@ public function getInsights(): array
     {
         return [
             UnusedInheritedVariablePassedToClosureSniff::class,
-            UnusedFunctionParameterSniff::class,
+            UnusedParameterSniff::class,
```

Comment:
Can you also update docs by changing metric badged here https://phpinsights.com/insights/code.html#unused-parameter 
and remove this part https://phpinsights.com/insights/code.html#unused-function-parameter ?